src/commcare_cloud/commands/ansible/run_module.py

```diff
@@ -22,6 +22,8 @@
     Argument('--become-user', help=(
         "run operations as this user (default=root)"
     ), include_in_docs=False),
+    Argument('--use-pem', action='store_true', help=(
+        "uses the pem file commcare_cloud_pem specified in public.vars")),
```

Comment:
I'm digging through this and I also don't see how the pem file is actually getting used (beyond the check for whether or not it exists.) but that step still runs without errors (and the output isn't showing me anything particularly weird). So I'm unsure why the pem file doesn't seem to be needed here. The bootstrap-users command is:
`ansible-playbook /Users/preethivaidyanathan/commcare-cloud/src/commcare_cloud/ansible/deploy_stack.yml -i /Users/preethivaidyanathan/commcare-cloud/environments/pvtest/inventory.ini -e @/Users/preethivaidyanathan/commcare-cloud/environments/pvtest/vault.yml -e @/Users/preethivaidyanathan/commcare-cloud/environments/pvtest/public.yml -e @/Users/preethivaidyanathan/commcare-cloud/environments/pvtest/.generated.yml --diff --tags=users -u ubuntu --forks 15 --ssh-common-args ''"'"'-o StrictHostKeyChecking=no'"'"''`

Reply:
But to address the rest of your comment, are you suggesting:

1. Checking for either `pem` or `root password` as opposed to only checking for pem like I'm doing now and
2. Create a separate command that has the `--use-pem` logic built into it (so the user never has to explicitly decide whether or not to use the pem file when writing a command line argument)

Reply:
I'm not sure why the pem file is not getting included in your run of `bootstrap-users` and things are still running fine. That's also confusing to me. I can offline with you and help get to the bottom of it if you'd like.

Yeah, that's what I'm suggesting: basically change `--use-pem` to `--use-factory-auth` and have that include pem or ask for root password as necessary. Preferably `bootstrap-users` would find a way to reuse the same code rather than having it implemented there as well.

Reply:
Cool, got it

Also, bootstrap-users does error without auth after all. I merged with master and ran it again today (without auth) and the errors were there. Really not sure what happened, but at least the thing that's happening now makes sense!